draco/core/containers.py

```diff
@@ -907,7 +952,10 @@ class SiderealStream(FreqContainer, VisContainer, SiderealContainer):
             "distributed_axis": "freq",
             "compression": COMPRESSION,
             "compression_opts": COMPRESSION_OPTS,
-            "chunks": (64, 256, 128),
+            "chunks": (128, 256, 512),
```

Comment:
I think broadly on the chunk sizes we should not bother chunking/compressing small files (~100 MB or less), as it's probably not worth the overhead. That's things like RFIMask, SystemSensitivity, etc. We should probably keep writing those out as HDF5 too, though I guess that's a pipeline config thing.

Reply:
I left some comments addressing this point in my review. More generally, I noted that this PR enables chunking by default for all datasets. Is their a benefit to doing this for datasets that we don't intend to compress? With it on by default we would need to specify those datasets where we don't want chunks. The way it was before you would specify only the datasets where you wanted chunking and compression. I guess which way we set the default should reflect how we intend the typical dataset to be written out.